src/nft/components/explore/Carousel.tsx

```diff
@@ -38,6 +34,7 @@ const CarouselItemCard = styled(a.div)`
   padding: 4px 12px 32px;
   position: absolute;
   will-change: transform;
+  width: calc(100%);
```

Comment:
So, this is interesting. `calc(100%)` evaluates to a width of the nearest relative parent. Thanks to that, we don't have to set card width manually later in the `render` method. As a result, we can remove listeners to window dimensions as well as setting a ref on a container.

Reply:
instead of calc render wouldn't just `100%` use the full width of the parent element?